### DISCUSSION TITLE
How to customize the theme / 修改主题

### DISCUSSION THREAD

**Discussion by Leader755** [vuepress-theme-hope/vuepress-theme-hope#609] [Q&A] [answered]
希望自己基于当前的主题，做一些适合自己的定制化开发。我好像不知道怎么去改。是直接克隆这个源码吗，但是跑不起来

I hope I can do some customized development that suits me based on the current theme. I don't seem to know how to change it. Is it to clone the source code directly? but it can't run

![image](https://user-images.githubusercontent.com/37180494/118397496-d88e2800-b686-11eb-8cb7-4691e37022cd.png)

**Comment by Mister-Hope** [comment, marked as answer]
The theme is totally written in TypeScript, which means all the source files need to be compiled first. So you should run `yarn build` first.

---

该主题完全用TypeScript编写，这意味着所有源文件都需要首先编译。 因此，您应该首先运行 `yarn build` 。

**Comment by Mister-Hope** [comment]
BTW, if you just want some style changes, we recommand you to use the [theme eject](https://vuepress-theme-hope.github.io/guide/eject/) feature to edit this theme.

---

顺便说一句，如果您只想更改样式，我们建议您使用[主题弹出](https://vuepress-theme-hope.github.io/zh/guide/eject/)功能来编辑此主题。

**Reply by Leader755** [reply]
好的我试下
